tree/dataframe/inc/ROOT/RDF/RInterface.hxx

```diff
@@ -1155,16 +1158,18 @@ public:
                                   ? ColumnNames_t()
                                   : ColumnNames_t(columnViews.begin(), columnViews.end());
 
-      // We build a default name and title based on the input columns
-      if (!(userColumns[0].empty() && userColumns[1].empty())) {
+      const auto validatedColumns = GetValidatedColumnNames(2, userColumns);
+
+         // We build a default name and title based on the input columns
+      if (!(validatedColumns[0].empty() && validatedColumns[1].empty())) {
          const auto v2Name_str = std::string(v2Name);
          const auto g_name = std::string(v1Name) + "*" + v2Name_str;
          graph->SetNameTitle(g_name.c_str(), g_name.c_str());
          graph->GetXaxis()->SetTitle(g_name.c_str());
          graph->GetYaxis()->SetTitle(v2Name_str.c_str());
       }
```

Comment:
`CreateAction` below already calls `GetValidatedColumnNames` internally...would it maybe be possible to move this logic inside `BuildAction<ActionTags::Graph>`, where column names have been correctly resolved?